docs/modules/ROOT/pages/usage.adoc

```diff
@@ -0,0 +1,47 @@
+= Usage
+
+The operator injects secret data into xref:volume.adoc[] mounts that declare a CSI volume with `driver: secrets.stackable.tech`.
+  
+A minimal secret-consuming `Pod` looks like this:
+
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: example-secret-consumer
+spec:
+  volumes:
+  - name: secret
+    csi:
+      driver: secrets.stackable.tech
+      volumeAttributes:
+        secrets.stackable.tech/class: tls
+        secrets.stackable.tech/scope: node,pod,service=secret-consumer-nginx
+  containers:
+  - name: ubuntu
+    image: ubuntu
+    stdin: true
+    tty: true
+    volumeMounts:
+    - name: tls
+      mountPath: /tls
+----
+
+xref:secretclass.adoc[] defines where the secrets come from. For example, the following `SecretClass`
+issues TLS certificates, storing its CA certificate in `secret-provisioner-tls-ca`:
```

Comment:
I might be dumb but it might be worthwhile saying that it's a Kubernetes `Secret` that it's being stored in. At least I assume so.

Reply:
Good point.